src/App.tsx

```diff
@@ -1,15 +1,27 @@
-import React from 'react';
+/* eslint-disable no-sequences */
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function filteredMovies(moviesServer: Movie[], mainstr: string): Movie[] {
+  const visibleMovies: Movie[] = moviesServer.filter((film: Movie) => film
+    .title.toLowerCase().includes(mainstr.toLowerCase().trim())
```

Comment:
mainstr.toLowerCase().trim() - the same on each iteration, do we need to lowercase it O(n) times?